.github/workflows/ci.yml

```diff
@@ -94,3 +94,12 @@ jobs:
       - uses: actions/checkout@v2
       - uses: dtolnay/rust-toolchain@clippy
       - run: cargo clippy --workspace --tests -- -Dclippy::all
+
+  clang-tidy:
+    name: Clang Tidy
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install clang-tidy
```

Comment:
You need to check out the code with
```
 - uses: actions/checkout@v2
```

Reply:
Right, this feels a bit silly now :sweat_smile:. I assumed you started from a checked out repository.